plugins/pca/tasks.py

```diff
@@ -40,36 +40,29 @@
 TASK_LOGGER = get_task_logger(__name__)
 
 
-def get_points(ent):
-    point = []
-    d = 0
-    while f"dim{d}" in ent.keys():
-        point.append(ent[f"dim{d}"])
-        d += 1
+def get_point(ent):
+    dimension_keys = list(ent.keys())
+    dimension_keys.remove("ID")
+    dimension_keys.remove("href")
+
+    dimension_keys.sort()
+    point = np.empty((len(dimension_keys, )))
```

Comment:
This looks a bit weird with the many () 
Also: is the comma inside the `len` function () intentional?